content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/go.md

```diff
@@ -89,23 +77,24 @@ To install and set up Orchestrion:
    go install github.com/DataDog/orchestrion@latest
    ```
    <div class="alert alert-info">Ensure <code>$(go env GOBIN)</code> (or <code>$(go env GOPATH)/bin</code>) is in your <code>$PATH</code>.</div>
```

Comment:
We had discussed this in the previous iteration... The issue is that how to actually do this can vary a lot depending on platform + other user settings... It ended up looking messy, and this really is just here to make sure the `orchestrion` command is indeed visible in the shell...

Also -- if you're going to be doing this in your CI/CD pipeline, you probably want to make the `$PATH` change at the same time as this is done (if necessary)... So having it here avoids folks forget about that step.

Reply:
Gotcha. Thanks for the clarification!